eng/source-build-patches/0004-Update-to-netstandard2.0.patch

```diff
@@ -0,0 +1,335 @@
+From 6babb13699adca5e18525cc600858a8ffb568ad4 Mon Sep 17 00:00:00 2001
+From: Omair Majid <omajid@redhat.com>
+Date: Thu, 5 Nov 2020 17:31:41 -0500
+Subject: [PATCH 2/4] Update to netstandard2.0
+
+---
+ ...soft.CodeAnalysis.BannedApiAnalyzers.Package.csproj |  4 ++--
+ ...alysis.PerformanceSensitiveAnalyzers.Package.csproj |  4 ++--
+ ...soft.CodeAnalysis.PublicApiAnalyzers.Package.csproj |  4 ++--
+ .../Roslyn.Diagnostics.Analyzers.Package.csproj        | 10 +++++-----
+ ...osoft.CodeAnalysis.CSharp.BannedApiAnalyzers.csproj |  4 +++-
+ .../Microsoft.CodeAnalysis.BannedApiAnalyzers.csproj   |  4 +++-
+ ...rosoft.CodeAnalysis.BannedApiAnalyzers.Setup.csproj |  4 ++--
+ ....CodeAnalysis.VisualBasic.BannedApiAnalyzers.vbproj |  4 +++-
+ ...nalysis.CSharp.PerformanceSensitiveAnalyzers.csproj |  4 +++-
+ ...t.CodeAnalysis.PerformanceSensitiveAnalyzers.csproj |  4 +++-
+ .../Core/Analyzers/DeclarePublicApiAnalyzer.Impl.cs    |  8 ++++----
+ .../Microsoft.CodeAnalysis.PublicApiAnalyzers.csproj   |  4 +++-
+ .../Core/CodeFixes/AnnotatePublicApiFix.cs             |  4 ++--
+ .../Core/CodeFixes/DeclarePublicApiFix.cs              |  4 ++--
+ ...ft.CodeAnalysis.PublicApiAnalyzers.CodeFixes.csproj |  4 +++-
+ ...rosoft.CodeAnalysis.PublicApiAnalyzers.Setup.csproj |  4 ++--
+ 16 files changed, 44 insertions(+), 30 deletions(-)
+
+diff --git a/nuget/Microsoft.CodeAnalysis.BannedApiAnalyzers/Microsoft.CodeAnalysis.BannedApiAnalyzers.Package.csproj b/nuget/Microsoft.CodeAnalysis.BannedApiAnalyzers/Microsoft.CodeAnalysis.BannedApiAnalyzers.Package.csproj
+index 140116f9d..6247917a1 100644
+--- a/nuget/Microsoft.CodeAnalysis.BannedApiAnalyzers/Microsoft.CodeAnalysis.BannedApiAnalyzers.Package.csproj
++++ b/nuget/Microsoft.CodeAnalysis.BannedApiAnalyzers/Microsoft.CodeAnalysis.BannedApiAnalyzers.Package.csproj
+@@ -1,7 +1,7 @@
+ ﻿<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+ <Project Sdk="Microsoft.NET.Sdk">
+   <PropertyGroup>
+-    <TargetFramework>netstandard1.3</TargetFramework>
++    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
netstandard1.3 is the correct target for this package. Builds using netstandard2.0 would not be correct.